dist/truncate.js

```diff
@@ -362,11 +362,23 @@
       ellipsis: '…',
       showMore: '',
       showLess: '',
-      position: 'end'
+      position: 'end',
+      lineHeight: 'auto'
     };
 
     this.options = $.extend({}, this._defaults, options);
 
+    if(this.options.lineHeight === 'auto') {
```

Comment:
@jeffchan I have added the space in the code. this option doesn't exist in jshint... maybe in eslint